pkg/kvstore/config.go

```diff
@@ -5,15 +5,16 @@ package kvstore
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"sync"
 
 	"github.com/cilium/cilium/pkg/logging/logfields"
 )
 
 var (
-	// selectedModule is the name of the selected backend module
-	selectedModule string
+	// ErrNotImplemented is the error which is returned when a functionality is not implemented.
+	ErrNotImplemented = errors.New("not implemented")
```

Comment:
nit: AFAIK this is only used in etcd_lease_test.go. I would move that error there now.